IHP/Environment.hs

```diff
@@ -1,4 +1,16 @@
 module IHP.Environment where
 import IHP.Prelude
 
+-- | The 'Environment' type is used to switch between development and production configurations.
+--
+-- You can set it with 'option Development' or 'option Production' in your Config.hs, or via the 'IHP_ENV' environment variable.
```

Comment:
You _can_, but you don't need to. When running `devenv up` it will set to DEV, and when you `deploy-to-nixos` it sets to Production.

I think we should clarify that, or maybe link to the docs